js/views/Portfolio.js

```diff
@@ -0,0 +1,91 @@
+export default class Portfolio {
+    constructor() {
+        this.title = "My Portfolio";
+        this.projects = [];
+
+        // 이미 인스턴스가 존재하는지 확인
+        if (!window.portfolioInstance) {
+            window.portfolioInstance = this;
+            this.fetchGitHubProjects();
+        } else {
+            return window.portfolioInstance;
+        }
+    }
```

Comment:
```suggestion
    static instance = null;
    
    constructor() {
        this.title = "My Portfolio";
        this.projects = [];

        // 싱글톤 패턴 개선
        if (Portfolio.instance) {
            return Portfolio.instance;
        }
        
        Portfolio.instance = this;
        this.fetchGitHubProjects();
    }
```

싱글톤으로 구현할때, window 외부 변수에 담기보다는 내부 변수로 처리해주는게 좋습니다.